Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:11 as builder
+
+COPY package.json package-lock.json ./
+
+RUN npm set progress=false && npm config set depth 0 && npm cache clean --force
+
+## Storing node modules on a separate layer will prevent unnecessary npm installs at each build
+RUN npm i && mkdir /ng-app && cp -R ./node_modules ./ng-app
+
+WORKDIR /ng-app
+
+COPY . .
+
+# Build the angular app in production mode and store the artifacts in dist folder
+RUN $(npm bin)/ng build --prod
```

Comment:
Looks like you're using this to build the prod angular app? I'm not sure if Netlify already handles this or not.

Reply:
Also I'm not sure if Netlify already handles all of this for us, at least in regards to the front end work